src/main/java/com/mitrais/chipper/temankondangan/backendapps/config/SecurityConfig.java

```diff
@@ -57,11 +57,12 @@ public HttpCookieOAuth2AuthorizationRequestRepository cookieAuthorizationRequest
     }
 
     private static final String[] AUTH_WHITELIST = {
-            "/swagger-resources/**",
-            "/swagger-ui.html",
-            "/v2/api-docs",
-            "/webjars/**",
-            "/register/**"
+//            "/swagger-resources/**",
+//            "/swagger-ui.html",
+//            "/v2/api-docs",
+//            "/webjars/**",
+//            "/register/**"
+            "/**"
```

Comment:
why it is commented? in my pc, i cannot access swagger if this line are commented